src/mappings/staking.ts

```diff
@@ -278,3 +286,78 @@ export async function handleWithdrawCancelled(
     await reportException('handleWithdrawCancelled', exception, event);
   }
 }
+
+async function updateStakeSummary(
+  event: EthereumLog<DelegationAddedEvent['args']>,
+  eraManager: EraManager | null
+): Promise<void> {
+  assert(event.args, 'No event args');
+  const { source, indexer, amount } = event.args;
+  const amountBn = amount.toBigInt();
+
+  const indexerEntity = await Indexer.get(indexer);
+  assert(indexerEntity, `Indexer ${indexer} does not exist`);
+  // const isFirstStake =
+  //   BigInt.fromJSONType(indexerEntity.totalStake.value) === BigInt(0);
+
+  let indexerStake = BigInt(0);
```

Comment:
for better readibility
```suggestion
  let newIndexerStake = BigInt(0);
```